elytron/src/test/resources/org/wildfly/extension/elytron/credential-security-factories.xml

```diff
@@ -16,6 +16,7 @@
                                    server="false"
                                    obtain-kerberos-ticket="true"
                                    debug="true"
+                                   mechanisms="KRB5 KRB5LEGACY"
```

Comment:
Is it meant you can use either `mechanisms` or `mechanism-oids`? If yes, could be `alternatives` relation configured in model? Or does it have sense to configure both of them at same time?

Also consider renaming `mechanisms` -> `mechanism-names` to be consistent to `mechanism-oids`